doc/theory/glossary.tex

```diff
@@ -396,54 +387,66 @@
 	
 	The name is inspired from the case when the persistence diagram comes from persistent homology.
 	
-	\subsection*{Distances, inner products and kernels} \label{metric_inner_product_and_kernel}
+	\subsection*{Metric space} \label{metric space}	
+	A set $X$ with a function	
+	\begin{equation*}	
+	d : X \times X \to \mathbb R	
+	\end{equation*}	
+	is said to be a \textit{metric space} if the values of $d$ are all non-negative and for all $x,y,z \in X$ ans $a \in \mathbb R$	
```

Comment:
Did you mean `and`?
Also, i can't find where `a` is used.

Reply:
Good catch!